### PR TITLE
MemQ Topic Creation Operator

### PR DESCRIPTION
**Pull request by ambud**
MemQ Topic Creation Operator

### CODE REVIEW

File: orion-server/src/main/java/com/pinterest/orion/core/automation/operator/memq/MemqTopicOperator.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import java.util.Map;
+
+import com.pinterest.orion.core.Attribute;
+import com.pinterest.orion.core.actions.memq.MemqTopicCreationAction;
+import com.pinterest.orion.core.automation.sensor.memq.MemqClusterSensor;
+import com.pinterest.orion.core.automation.sensor.memq.MemqTopicRepoSensor;
+import com.pinterest.orion.core.automation.sensor.memq.TopicConfig;
+import com.pinterest.orion.core.memq.MemqCluster;
+
+import jersey.repackaged.com.google.common.collect.Sets;
```
Comment (kabochya) [inline orion-server/src/main/java/com/pinterest/orion/core/automation/operator/memq/MemqTopicOperator.java:15]: Probably use the guava dependency

File: orion-server/src/main/java/com/pinterest/orion/core/memq/MemqCluster.java [hunk 2]
```diff
@@ -65,12 +67,15 @@ public MemqCluster(String id,
...
 
+  @SuppressWarnings("unchecked")
   @Override
   protected void bootstrapClusterInfo(Map<String, Object> config) throws PluginConfigurationException {
     this.config = config;
     setAttribute(ZK_CONNECTION_STRING, config.get(ZK_CONNECTION_STRING));
     setAttribute(SERVERSET_PATH, config.get(SERVERSET_PATH));
     setAttribute(CLUSTER_INFO_DIR, config.get(CLUSTER_INFO_DIR));
+    notificationClusterConfig = (Map<String, String>) config
+        .get(MemqTopicCreationAction.NOTIFICATION_CLUSTER_CONFIG);
```
Comment (kabochya) [inline orion-server/src/main/java/com/pinterest/orion/core/memq/MemqCluster.java:78]: Should we move this constant `MemqTopicCreationAction.NOTIFICATION_CLUSTER_CONFIG` from the action to the cluster instead? It's seems to used by multiple components.

**Review by kabochya — APPROVED** [review]
Few nits, otherwise lgtm

File: orion-server/src/main/java/com/pinterest/orion/core/automation/operator/memq/MemqTopicOperator.java (hunk 1 above)
Comment (ambud) [reply orion-server/src/main/java/com/pinterest/orion/core/automation/operator/memq/MemqTopicOperator.java:15]: Good catch, fixed.

File: orion-server/src/main/java/com/pinterest/orion/core/memq/MemqCluster.java (hunk 2 above)
Comment (ambud) [reply orion-server/src/main/java/com/pinterest/orion/core/memq/MemqCluster.java]: Fixed

**Pull request merged**